examples/data-objects/simple-fluidobject-embed/src/index.tsx

```diff
@@ -58,6 +58,9 @@ export const SimpleFluidObjectEmbedInstantiationFactory =
         SimpleFluidObjectEmbed,
         [],
         {},
+        new Map([
+            ClickerInstantiationFactory.registryEntry,
```

Comment:
Nice :)  Could you also please remove the `ClickerInstantiationFactory` from the registry of the `ContainerRuntimeFactoryWithDefaultDataStore` below (looks like the entry was misplaced more or less).

Reply:
Oh, now I get it :) Just commited the change.
So this basically means `ContainerRuntimeFactoryWithDefaultDataStore` should have the registry entry of its "root" Fluid Data Object instantiation factory and all the "child" Fluid Data Object's registry entries should be provided in the "root"'s instantiation factory (here `SimpleFluidObjectEmbedInstantiationFactory`)?

Reply:
Exactly!